src/tests/server/routing/v1/projects/middleware.test.js

```diff
@@ -21,11 +22,13 @@ const FORCE_CLEANUP = true;
 
 describe(`project middlerware tests`, async () => {
   before(async () => await initTestDatabase());
+  beforeEach(() => clearTestData());
   after(() => {
     concludeTesting();
     closeReader();
   });
 
+  // NOTE: This test has flaked 2 times
```

Comment:
Not sure how we want to handle flakey tests, but I figured I'd add a note here and if it keeps flaking we can remove it.

Reply:
You got it!